lib/workers/repository/changelog/index.ts

```diff
@@ -20,8 +22,12 @@ export async function embedChangelogs(
   await pMap(branches, embedChangelog, { concurrency: 10 });
 }
 
-export function needsChangelogs(upgrade: BranchUpgradeConfig): boolean {
-  for (const field of exposedConfigOptions) {
+export function needsChangelogs(
+  upgrade: BranchUpgradeConfig,
+  fields = exposedConfigOptions.filter((o) => o !== 'commitBody')
+): boolean {
+  // commitBody is now compiled when commit is done
+  for (const field of fields) {
     // fields set by `getChangeLogJSON`
     if (containsTemplates(upgrade[field], ['logJSON', 'releases'])) {
       return true;
```

Comment:
Should this be a warn now?

Reply:
it's here
https://github.com/renovatebot/renovate/blob/6fe3f2ff326ed4136a6a1d529407997a55815c03/lib/workers/repository/updates/branchify.ts#L79-L88

Reply:
i need this fuction later here

https://github.com/renovatebot/renovate/blob/6fe3f2ff326ed4136a6a1d529407997a55815c03/lib/workers/repository/update/branch/index.ts#L482-L499